core/src/main/java/com/spotify/metrics/core/SemanticMetricRegistry.java

```diff
@@ -383,23 +392,35 @@ public void removeListener(final SemanticMetricRegistryListener listener) {
      * Returns a map of all the deriving meters in the registry and their names which match the
      * given filter.
      *
-     * @param filter    the metric filter to match
+     * @param filter the metric filter to match
      * @return all the deriving meters in the registry
      */
     public SortedMap<MetricId, DerivingMeter> getDerivingMeters(final SemanticMetricFilter filter) {
         return getMetrics(DerivingMeter.class, filter);
     }
 
     /**
-     * Atomically adds the given metric to the set of metrics.
+     * Returns a map of all the distributions metrics in the registry and their
+     * names which match the given filter.
      *
+     * @param filter the metric filter to match
+     * @return a sorted Map of distribution metrics
+     */
+    public SortedMap<MetricId, Distribution> getDistribution(final SemanticMetricFilter filter) {
```

Comment:
It should be **getDistributions** as other methods

Reply:
Done